app/views/catalog/_index.html.erb

```diff
@@ -63,7 +63,10 @@
       <%= render 'additional_results_block' %>
     <%- end %>
 
-    <%= render 'results_pagination' %>
+    <%- if !@response.empty? %>
+      <%= render 'shared/side_nav_minimap' %>
```

Comment:
Is this particular position in the DOM of this element important for keyboard accessibility purposes?  Might be a question for @jvine  (or perhaps you all have already worked this out).